nx/lib/nx.ex

```diff
@@ -4396,6 +4396,39 @@ defmodule Nx do
         ]
       >
 
+  ## Containers
```

Comment:
This is a too verbose doctest. I think I would do a basic test with a two element tuple and leave the rest to actual test files. :)